src/unix/fs.c

```diff
@@ -1076,16 +1076,12 @@ static ssize_t uv__fs_copyfile(uv_fs_t* req) {
   bytes_to_send = src_statsbuf.st_size;
   in_offset = 0;
   while (bytes_to_send != 0) {
-    err = uv_fs_sendfile(NULL,
-                         &fs_req,
-                         dstfd,
-                         srcfd,
-                         in_offset,
-                         bytes_to_send,
-                         NULL);
+    uv_fs_sendfile(NULL, &fs_req, dstfd, srcfd, in_offset, bytes_to_send, NULL);
     uv_fs_req_cleanup(&fs_req);
-    if (err < 0)
+    if (fs_req.result < 0) {
```

Comment:
Here we are accessing `fs_req` after cleaning it up. It doesn't seem like a safe thing to do, we might `memset` the thing to 0 or anything in debug mode, for example. (Not saying we do it now, but we might decide to do that one day)